corehq/apps/userreports/fixtures.py

```diff
@@ -50,7 +54,14 @@ def __call__(self, user, version, last_sync=None):
             return []
 
         root = ElementTree.Element('fixture', attrib={'id': self.id})
-        reports_elem = ElementTree.Element('reports')
+        reports_elem = ElementTree.Element(
+            'reports',
+            attrib={
+                'last_sync': ServerTime(datetime.utcnow()).user_time(
+                    user.get_time_zone() or pytz.utc
```

Comment:
More a question for @dannyroberts but I'm curious, would make sense for `user_time` to default to UTC if the given time zone is null?